spynnaker/pyNN/models/neuron/synaptic_manager.py

```diff
@@ -1219,3 +1496,19 @@ def changes_during_run(self):
         if self.__synapse_dynamics is None:
             return False
         return self.__synapse_dynamics.changes_during_run
+
+
+class _AppKeyInfo(object):
+
+    __slots__ = ["app_key", "app_mask", "core_mask", "core_shift", "n_neurons"]
```

Comment:
default 0's for core mask, core shift, n neurons

Reply:
Not now needed because of a different solution